float64[] position_list
float64[] velocity_list
int64[] timestamp_list


